index.js

```diff
@@ -0,0 +1,53 @@
+const list = ['掃除', '買い物', '散歩'];
+const display_text = `現在持っているタスク一覧\n`;
+function list_display() {
+  console.log(display_text);
+  for (let i = 0; i < list.length; i++) {
+    console.log(`${i}:${list[i]}`);
+  }

```

Comment:
`for` 文でもいいですが、今のうちに `forEach` に慣れておいてもいいかなと思いました。

```suggestion
  list.forEach((task, index) => {
    console.log(`${index}:${task}`);
  });
```